presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewQueryOptimizer.java

```diff
@@ -105,35 +107,28 @@
         extends AstVisitor<Node, Void>
 {
     private static final Logger log = Logger.get(MaterializedViewQueryOptimizer.class);
-
-    private static final QualifiedName MIN = QualifiedName.of("MIN");
-    private static final QualifiedName MAX = QualifiedName.of("MAX");
     private static final QualifiedName SUM = QualifiedName.of("SUM");
     private static final QualifiedName COUNT = QualifiedName.of("COUNT");
-    private static final Set<QualifiedName> SUPPORTED_FUNCTION_CALLS = ImmutableSet.of(MIN, MAX, SUM, COUNT);
```

Comment:
These functions are duplicated in both places. Can we centralized them in `MaterializedViewUtils` and make them public?